src/video/SDL_video_c.h

```diff
@@ -55,4 +55,6 @@ extern int SDL_VideoInit(const char *driver_name);
  */
 extern void SDL_VideoQuit(void);
 
+extern int SDL_SetVSyncWindowTexture(SDL_Window *window, int vsync);
```

Comment:
Can you change the name to SDL_SetWindowTextureVSync()?

Reply:
done!